docker/README.md

```diff
@@ -1,9 +1,27 @@
+# Launch New Instance
+
+1. Use "Launch instance from template"
+Use the `alchemiscale-template` to lauch a new EC2 instance.
+The only setting that will need modification is under "Key pair (login)".
+Select "Proceed without a key pair" (SSH keys have already be added to the image).
+
+2. Setup DNS
+Copy the public IP from the newly launched instance.
+Use that IP address to create an A record with desired hostname.
+
+3. SSH into the instance.
+Use the hostname to ssh into the instance to test the DNS.
+The username is `ec2-user`.
+For example: `ssh ec2-user@compute.alchemiscale.org`
+
 # Deployment via `docker-compose`
 
 Note: this assumes a running Docker daemon and the `docker-compose` plugin.
 
 To deploy (using bash + EC2 host):
 
+0. `cd` into the correct directory `cd /home/ec2-user/alchemiscale/docker`
```

Comment:
Can't comment on the right line but I think it might also be worth mentioning that you need to add `HOST_DOMAIN`  equal to whatever host domain you set in the A records and `ALCHEMISCALE_DOCKER_IMAGE` to be a recent image off `ghcr.io` the default one in `.env.testing` is an old one off a feature branch.